packages/vrplib/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: vrplib
+  version: 2.0.0
+  top-level:
+    - vrplib
+source:
+  url: https://files.pythonhosted.org/packages/f4/a6/3cd94acf14036cee52418b4701caaf7bd0e0f619611648deb41725693e29/vrplib-2.0.0.tar.gz
```

Comment:
vrplib is a pure python library. Could you use a wheel file directly instead of using the tar.gz archive?